localstack/services/route53resolver/models.py

```diff
@@ -13,3 +14,73 @@ def __init__(self):
         self.firewall_domain_lists = {}
         self.firewall_domains = {}
         self.firewall_rules = {}
+        self.firewall_rule_group_associations = {}
+
+    def get_firewall_rule_group(self, id):
```

Comment:
I would keep try to keep these utility functions outside of the backend (at top-module level). 
This to keep the backends as small and simplest as possible since we need to pickle them to support some advanced features.

Reply:
Shifted to the top level.